test/integration/sagemaker/test_spark.py

```diff
@@ -361,6 +361,48 @@ def test_sagemaker_java_jar_multinode(tag, role, image_uri, configuration, sagem
     assert len(output_contents) != 0
 
 
+def test_sagemaker_java_jar_singlenode(tag, role, image_uri, configuration, sagemaker_session, sagemaker_client):
```

Comment:
Is the only difference between this test and the `multinode` test the `instance_count`? If so, could you parameterize the instance_count so that the test code isn't duplicated between those tests?

Reply:
Done